addons/web/__manifest__.py

```diff
@@ -224,6 +224,7 @@
             'web/static/src/env.js',
             'web/static/src/core/utils/transitions.scss',  # included early because used by other files
             'web/static/src/core/**/*',  # Note that 'web/static/src/core/utils/ui.js' is included in assets_frontend_minimal already
+            'web/static/src/views/fields/badge_selection/badge_selection.scss',
```

Comment:
This import seems no longer needed, right? 🤔 

The classes defined in that file is no longer used in project_task_templates xml file. 🤔